### PR TITLE
Sprint: Fix stamina start value

### PR DESCRIPTION
**Pull request by ZenBre4ker**
#1146

### CODE REVIEW

File: gamemodes/terrortown/gamemode/shared/sh_player_ext.lua [hunk 1]
```diff
@@ -23,7 +23,7 @@ end
 function plymeta:SetupDataTables()
 	-- This has to be transferred, because we need the value when predicting the player movement
 	-- It turned out that this is the only reliable way to fix all prediction errors.
-	self:NetworkVar("Float", 0, "SprintStamina")
+	self:NetworkVar("Float", 1, "SprintStamina")
```
Comment (Histalek) [inline gamemodes/terrortown/gamemode/shared/sh_player_ext.lua:26]: This only changes the slot of the network var, not the default/starting value

**Review by Histalek — CHANGES_REQUESTED** [review]

**Comment by ZenBre4ker** [comment]
Yeah now I actually tested it 😄

**Comment by TimGoll** [comment]
Is 1 the max value? Aren't you able to set the stamina in the UI to something greater than 1?

**Comment by ZenBre4ker** [comment]
https://github.com/TTT-2/TTT2/blob/master/gamemodes/terrortown/gamemode/shared/sh_sprint.lua#L62-L74

As far as I understqand the code, 1 is max. The rest is just appropriately scaled I guess.

**Comment by TimGoll** [comment]
I agree:

https://github.com/TTT-2/TTT2/blob/master/gamemodes/terrortown/gamemode/shared/sh_sprint.lua#L54

**Review by TimGoll — APPROVED** [review]

**Review by Histalek — APPROVED** [review]

**Pull request merged**